apps/services/contentful-entry-tagger/infra/contentful-entry-tagger-service.ts

```diff
@@ -21,13 +21,9 @@ export const serviceSetup =
           },
           paths: ['/'],
           extraAnnotations: {
-            dev: {
-              'nginx.ingress.kubernetes.io/enable-global-auth': 'false',
-            },
-            staging: {
-              'nginx.ingress.kubernetes.io/enable-global-auth': 'false',
-            },
-            prod: { 'nginx.ingress.kubernetes.io/enable-global-auth': 'false' },
+            dev: {},
+            staging: {},
+            prod: {},
```

Comment:
What does this mean? Contentful webhooks are calling both the entry tagger and search indexer endpoints. We want that to still work (otherwise the user management in Contentful will degrade and content will stop updating for the search)

Reply:
@AndesKrrrrrrrrrrr / @busla was this addressed?